standard/expressions.md

```diff
@@ -3888,7 +3914,7 @@ Unless one of these conditions is true, a binding-time error occurs.
 <!-- markdownlint-disable MD028 -->
 
 <!-- markdownlint-enable MD028 -->
-> *Example*: The following example checks whether an argument of an unconstrained type parameter type is `null`.
+> *IncompleteExample*: The following example checks whether an argument of an unconstrained type parameter type is `null`.
```

Comment:
I don't think this was where you intended to insert `Incomplete`.

Reply:
agreed; removed